test/test_twodturb.jl

```diff
@@ -1,5 +1,4 @@
 import FourierFlows.TwoDTurb
-import FourierFlows.TwoDTurb: energy, enstrophy, dissipation, work, drag
```

Comment:
This line was removed because in tests we can't import functions from `TwoDTurb` because then there is conflict with imports of same-name-functions from `BarotropicQG`.